doc/nrf/releases/release-notes-latest.rst

```diff
@@ -24,7 +24,7 @@ The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repositor
 
```

Comment:
Probably need to update the commit in line 23?

Reply:
Yes, definitely. I will do that